File: repaso objetos/python/PhytonEst/Persona.py
class Persona:

    def __init__(self):
        self.__nombre__ = ""
        
    def set__nombre__(self, nombre=""):
        self.__nombre__ = nombre

    def presentar(self):
        print("Hola mi nombre es "+ self.__nombre__)    

r=Persona()
r.set__nombre__("Miguel")

r.presentar()

File: repaso objetos/java/Conciertopf/Fabrica_especial2/Persona.java
package Conciertop.Fabrica_especial2;
public class Persona {
    private String nombre;

    public void setNombre(String nombre){
        this.nombre=nombre;
    }
    public void presentar(){
        System.out.println("Hola mi nombre es "+this.nombre);
    }
}
